GitUI/UserControls/FileStatusList.cs

```diff
@@ -72,8 +72,11 @@ public FileStatusList()
             FileStatusListView.SmallImageList = _images;
             FileStatusListView.LargeImageList = _images;
 
-            NoFiles.Visible = false;
+            HandleVisibility_NoFilesLabel_FilterComboBox(filesPresent: true);
+            this.Controls.SetChildIndex(NoFiles, 0);
             NoFiles.Font = new Font(SystemFonts.MessageBoxFont, FontStyle.Italic);
+
+            _filter = new Regex(".*");
```

Comment:
If we assign to null, then we need to explicitly check for it. While new Regex(".*") accepts all files already.
I prefer avoiding null values for logical checks if it is possible.